pkg/referrerstore/oras/authprovider/aws/awsecrbasic_test.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright The Ratify Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package aws
+
+import (
+	"context"
+	"encoding/base64"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/ecr/types"
+	"strings"
+	"testing"
+	"time"
+)
+
+const (
+	testUsername = "AWS"
+	testPassword = "eyJwYXlsb2FkIjoiOThPNTFqemhaUmZWVG"
+	testProxy    = "PROXY_ENDPOINT"
+)
+
+// Verifies that userName, password, and
+// environment variables are properly set
```

Comment:
not super critical but I think the current test doesn't test the environment variable check logic in `getEcrAuthToken`? I'm if we don't add any, but we can then update the comment accordingly.

Reply:
Looking...

Reply:
Updated comments and func name. Tested OK locally.
```
go test -v -run TestAwsEcrBasicAuthProvider_Enabled
=== RUN   TestAwsEcrBasicAuthProvider_Enabled
--- PASS: TestAwsEcrBasicAuthProvider_Enabled (0.00s)
PASS
ok  	github.com/deislabs/ratify/pkg/referrerstore/oras/authprovider/aws	0.528s
```